README.md

```diff
@@ -62,105 +62,67 @@ For running integration tests in Anvil node, run `make anvil-tests`. This runs t
-       * `forge script script/<SCRIPT_NAME> --rpc-url $RPC_URL --broadcast --verify -vvvv`
-       
-          Example: `forge script script/001_DeployPluginManager.s.sol --rpc-url $RPC_URL --broadcast --verify -vvvv`
+### Deploy & Verify EPv0.6 SponsorPaymaster
+Deploy, setup and verify SponsorPaymaster compatible with ERC-4337 v0.6.
 
-      Tip: before executing the above command, verify the simulation works as expected by running the above command without the `--broadcast` and `--verify` flags. This way, you can also make sure your address will have enough tokens to cover the transaction fee estimated in the simulation.
+#### Run deploy command
+Note that deploy commands attempt to deploy on all chains listed in foundry.toml.
 
-      Note: if you are deploying on a local network, don't use the `--verify` flag.
-    
-  4. Include the relevant logs from the `broadcast` folder in your commit.
+```shell
+forge script script/bytecode-deploy/107_DeploySponsorPaymasterEPv06.s.sol  -vvvv --slow --broadcast --force --multi
+```
 
-        Tip: if you did multiple runs, search the appropriate block explorer for the tx hash corresponding the desired contract's deployment, and then search the logs for the transaction hash.
+#### Setup SponsorPaymaster
 
-        Tip: logs are organized by chain ID in the lower levels in the `broadcast` folder. Use <https://chainlist.org/> to lookup IDs of common chains.
+Need to use block explorer to run `addStake` and `setVerifyingSigner` from owner address.
 
-  5. Create or update the corresponding file in the `script/cmd` folder using the creation bytecode of the contract from the logs. See the below "Chain Expansion" section for details on how to format of the files in the `script/cmd` folder.
-  6. Verify in block explorer like etherscan using standard input json
-       * Create the standard input json: run the beoow command
-       
-          ```shell
-          forge verify-contract <contract_address> <relative_path_to_source:classname> --show-standard-json-input > <script/verify/<filename>
+#### Verify SponsorPaymaster
 
-          # Example
-          forge verify-contract 0x03431fb00fb2e26b5bc502dfef8da30e1c8643b8 src/msca/6900/v0.7/plugins/v1_0_0/utility/DefaultTokenCallbackPlugin.sol:DefaultTokenCallbackPlugin --show-standard-json-input > script/verify/DefaultTokenCallbackPlugin.json
-          ```
-       * Verify and publish in block explorer (etherscan example)
-         * Compiler type: `Solidity (Standard-Json-Input)`
-         * Compiler version: `v0.8.24`
-         * License: Option 5 in https://etherscan.io/contract-license-types
-         * Upload the JSON file
-         * If the contract you are verifying took constructor arguments, input the output of the below command with the `0x` prefix removed into the "Constructor Arguments ABI-encoded" section:
+Use `script/bytecode-deploy/standard-json-input/SponsorPaymasterEP.json` to verify. Select the following options
```

Comment:
minor, should probably use [SponsorPaymaster.json](https://github.com/circlefin/buidl-wallet-contracts/blob/entry-point-v0.6/script/bytecode-deploy/standard-json-input/SponsorPaymaster.json)